src/net/chat.rs

```diff
@@ -318,6 +334,9 @@ pub struct ConversationP2P {
 
     pub stats: NeighborStats,
 
+    // which stacker DBs this peer replicates
```

Comment:
Should be a rustdoc, and should also explain that StackerDBs can be identified by ContractId

Reply:
Fixed